Code/Framework/AzFramework/AzFramework/Spawnable/SpawnableEntitiesManager.h

```diff
@@ -132,6 +135,14 @@ namespace AzFramework
             EntitySpawnTicket::Id m_ticketId;
             uint32_t m_requestId;
         };
+        struct DespawnEntityCommand
+        {
+            EntityDespawnCallback m_completionCallback;
+            Ticket* m_ticket;
+            EntitySpawnTicket::Id m_ticketId;
+            uint32_t m_requestId;
+            AZ::EntityId m_entityId;
```

Comment:
Nitpick: very nitpicky, but since we discussed this offline, `m_entityId` should be put before `m_ticketId` because `m_entityId` is 64-bit and `m_ticketId` is a 32-bit number. (It's not going to make any difference for the struct size here.)